Packs/ShadowxSOCAI/Integrations/ShadowxSOCAI/ShadowxSOCAI.yml

```diff
@@ -0,0 +1,225 @@
+commonfields:
+  id: ShadowxSOCAI
+  version: -1
+name: ShadowxSOCAI
+display: ShadowX SOCAI
+category: Analytics & SIEM
+description: ShadowX SOCAI filters sensitive data in security logs, analyzes them with GenAI (e.g. ChatGPT, Gemini, Claude), and returns enriched guidance such as AI response, recommendation, risk/result classification, and prediction score. Supports API Key (JSON /Api/ endpoints) and cookie (HTML) modes.
+fromversion: 6.10.0
+
+configuration:
+# Base URL
+- display: Server URL (e.g., https://app.shadowx.ai)
+  name: url
+  type: 0
+  required: true
+
+# API Key (Bearer) mode
+- displaypassword: API Key (recommended). If set, email/password are ignored.
+  name: credentials_api
+  type: 9
+  hiddenusername: true
+  required: false
+
+- display: API Submission Path (optional)
+  name: api_submission_path
+  type: 0
+  required: false
+  defaultvalue: /Api/SecurityTasks/Create
+
+- display: API Check Path Format
+  name: api_check_path_format
+  type: 0
+  required: false
+  defaultvalue: /Api/SecurityTasks/Details?taskID={task_id}
+  additionalinfo: "Must include {task_id} (e.g., /Api/SecurityTasks/Details?taskID={task_id})"
```

Comment:
both these params are api calls related paths, not related to the user of the integration, please remove and have them as hard coded values in the related commands